modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/js/ViewItemSelectorViewDescriptor.ts

```diff
@@ -49,20 +49,38 @@ export default function ({
 		});
 	};
 
+	const onBlurHandler = delegate(
+		document.getElementById(`${namespace}entriesContainer`)!,
+		'blur',
+		'tbody tr',
+		(event: DelegatedEvent<any>) => {
+			event.target.classList.remove('table-focus');
+		}
+	);
+
 	const onClickHandler = delegate(
 		document.getElementById(`${namespace}entriesContainer`)!,
 		'click',
-		'.entry',
+		'tbody tr',
```

Comment:
Hey @pat270, 
my only concern is if this will work in other views. In table view, which is the one that your screenshot shows, this works, but could you check with card or descriptive view? Or please provide me the steps to test this? 

From my understanding, it will probably fail. `.entry` class is added for each view but `'tbody tr'` only refers to the table view.

Reply:
@ambrinchaudhary You're right, it does break something for card view in add new basic web content > related assets. Click the select button shown on the screen shot.

![related-assets](https://github.com/user-attachments/assets/2f8e2b3c-f616-4276-bdc0-3a5d8171fdd5)

I added the classes `entry entry-selector` on the table row if it's list view and not multiple selection. You can get to where we are fixing it by going to web content > templates > new > structure plus button.

![new-template-structure](https://github.com/user-attachments/assets/cc1a088a-3fa0-41e1-8bd3-30e2e109e185)

It should open this modal.

![modal](https://github.com/user-attachments/assets/42b49fb7-57e7-4b35-a3ea-76d3fad08310)

Reply:
Perfect, thank you @pat270!